tools/test/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+load("@aspect_bazel_lib//lib:jq.bzl", "jq")
+
+jq(
+    name = "final_package_json",
+    # Note: this jq filter relies on the order of the inputs
+    # buildifier: do not sort
+    srcs = [
+        "root_package.json",
+        "project_package.json",
+    ],
+    args = [
+        "--slurp",
+    ],
+    filter_file = "//tools:package_json_release_filter.jq",
+)
+
+# Test correctness of the filter that prepares each project's package.json file for release
+diff_test(
+    name = "package_json_filter_test",
+    file1 = "expected_package.json",
+    file2 = select({
+        # jq outputs CR on windows https://github.com/stedolan/jq/issues/92
+        # strip the CRs to do a correct comparison
+        "@bazel_tools//src/conditions:host_windows": ":final_package_json_cr_stripped",
+        "//conditions:default": ":final_package_json",
```

Comment:
Also, why not strip `\r` always? that should avoid the select + another target.

Reply:
I was thinking it would be more self documenting, but yeah I agree it would be simpler to leave out the select.